xbmc/AppParamParser.cpp

```diff
@@ -35,6 +35,8 @@
 #endif
 #include <stdlib.h>
 
+using namespace KODI::MESSAGING;
```

Comment:
AppMessenger is used if you want something executed by the main thread. AE, VDPAU, VAAPI, dvdplayer have messaging as well but are not ties to this namespace. This is mainthread only and should be flagged as such.